examples/dynamics/absolutely inelastic_collision.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+
+# Inelastic collision transforms two objects with masses m1, m2 and velocities v1 and v2 to one object with mass m = m1+m2 and unknown velocity to be resolved
+#assert 1-dimentional env
+
+from symplyphysics import (
+    units, convert_to, SI
+)
+from symplyphysics.definitions   import  momentum_is_mass_times_velocity as momentum_defined
+from symplyphysics.laws.dynamics import  momentum_after_collision_equals_to_momentum_before as momentum_law
+
+m1 = units.Quantity('m1')
+v1 = units.Quantity('v1')
+m2 = units.Quantity('m2')
+v2 = units.Quantity('v2')
+m = units.Quantity('m3')
+
+#The bullet with mass of 10gram and velocity of 700m/s hith the steady 75kg piece of plasticine, stucks in it and makes this plasticine move with some unknown velocity.
+
+SI.set_quantity_dimension(m1, units.mass)
+SI.set_quantity_dimension(m2, units.mass)
+SI.set_quantity_dimension(m, units.mass)
+
+SI.set_quantity_dimension(v1, units.velocity)
+SI.set_quantity_dimension(v2, units.velocity)
+
+SI.set_quantity_scale_factor(m1, 0.01 * units.kilogram)
```

Comment:
You can use dimensions exactly as in description:
```suggestion
SI.set_quantity_scale_factor(m1, 10 * units.gram)
```